src/components/DetailViewHeader.jsx

```diff
@@ -110,20 +131,39 @@ export class DetailViewHeader extends React.PureComponent {
                 canManageUsers={true}
               />
             )}
-            <Button className={classes.exportButton} variant="contained" color="primary">
-              <a
-                target="_blank"
-                rel="noopener noreferrer"
-                href={`${terraUrl}/#import-data?url=${window.location.origin}&snapshotId=${of.id}&snapshotName=${of.name}&format=snapshot`}
+            <Dialog open={isOpen}>
+              <DialogContent>
+                <Typography variant="h5">Your data snapshot is being exported</Typography>
+                {/* TODO: Make this loading state more descriptive */}
+                <div className={clsx(classes.centered, classes.content)}>
+                  <CircularProgress />
+                </div>
+              </DialogContent>
+            </Dialog>
+            <Tooltip
+              title="Exporting a snapshot to a workspace means that all members of your workspace will be able to have read only access to the tables and files in the snapshot"
```

Comment:
I think this is different than what Lou had in the mocks...is that ok with him?

Reply:
The tooltip text? It should match the latest mocks: https://broadworkbench.atlassian.net/browse/DR-2403?focusedCommentId=55502

Reply:
Oh sorry, I meant generally having a dialog